Makefile

```diff
@@ -83,7 +83,7 @@ build: bundles
 	docker build -t "$(DOCKER_IMAGE)" .
 
 docs-build:
-	git diff --name-status upstream/release..upstream/docs docs/ > docs/changed-files
+	( git remote | grep -v upstream ) || git diff --name-status upstream/release..upstream/docs docs/ > docs/changed-files
```

Comment:
mmm, I was thinking to change this to detecting what the user had aliased the `github.com/docker/docker` repo to